### PR TITLE
Disable color-contrast for clinical-lowlight-theme wdio testing

### PR DESCRIPTION
**Pull request by StephenEsser**
### Summary

This PR disables the axe-core color-contrast ratio rule for lowlight theme WDIO testing. The lowlight theme does not pass the default color contrast and follows a slightly different standard.

### Additional Details

I logged an issue to axe-core about adjusting the rule settings. The initial feedback seems promising.

https://github.com/dequelabs/axe-core/issues/2175

I spent a few hours attempting to write tests for this, but mocking the process.env seems to be far more challenging than it sounds. I attempted to clear the cache per test run, but was not able to get anything working. The process.env is set the first time the file is imported and every import thereafter uses the cached values.

@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: config/webpack/postcss/ThemePlugin.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const SUPPORTED_THEMES = [
   'orion-fusion-theme',
   'clinical-lowlight-theme',
   'cerner-clinical-theme',
+  'terra-default-theme',
```
Comment (StephenEsser) [inline config/webpack/postcss/ThemePlugin.js:11]: This isn't really doing anything, I added it here more or less as a "We acknowledge the default theme as a theme and this is the name". The theme plugin still needs to filter through the scoped themes and remove any unused themes.

This becomes useful when a project has set a different them as the default theme in the theme config file but still wants to run tests against the terra-default-theme.

Comment (emilyrohrbough) [inline config/webpack/postcss/ThemePlugin.js:11]: will you be opening another PR to filter out the terra-default-theme when the screenshots are generated during wdio in the visual-regression run?

Comment (StephenEsser) [reply config/webpack/postcss/ThemePlugin.js:11]: I'm not sure I understand the scenario where the default theme will need to be filtered out. Can you explain the scenario and expectations a little more?

Comment (emilyrohrbough) [reply config/webpack/postcss/ThemePlugin.js:11]: I'm thinking when the default theme is used, but isn't the default/root theme of the app...and I actually there is more to it this than I was originally thinking before I typed this out. 

My thoughts are this, if a project used the `clinical-lowlight-theme` as the default theme, their configuration might look like this:
```js
const themeConfig = {
  theme: 'clinical-lowlight-theme',
  scoped: ['terra-default-theme'],
};
module.exports = themeConfig;
```
When they want to run wdio tests, the can run as is, without passing in the theme via `THEME` because the theme configuration handles specify the `clinical-lowlight-theme` as the theme that should be the theme used during testing.

```sh
npm run wdio
```

to generate screenshots like:
```
/wdio/_snapshots_/reference/clinical-lowlight-theme/en        <-- does this work correctly when clinical-lowlight-theme is default theme? It might not...
```

Then, they will generate screenshots for the `terra-default-theme` because their app also supports it:

```sh
THEME='terra-default-theme` npm run wdio
```
currently today, this we generate screenshots like:
```
/wdio/_snapshots_/reference/en        <-- no theme folder...prob needs to in next MVB 
```
however, by defining `THEME='terra-default-theme`, this actually outputs like:
```
/wdio/_snapshots_/reference/terra-default-theme/en        <-- inconsistent
```
Also, neither the ThemePlugin & aggregate-themes script account for the `terra-default-theme`, which will result in no aggregation...Maybe the Theme Plugin if it didn't filter out the default theme. (@mjhenkes does the fallback get filtered out or just the additional themes?)

Comment (StephenEsser) [reply config/webpack/postcss/ThemePlugin.js:11]: I'm having a little trouble following what you are expecting to happen vs what happens today.

## Section 1

> My thoughts are this, if a project used the clinical-lowlight-theme as the default theme, their configuration might look like this:

```js
const themeConfig = {
  theme: 'clinical-lowlight-theme',
  scoped: ['terra-default-theme'],
};
module.exports = themeConfig;
```

> When they want to run wdio tests, the can run as is, without passing in the theme via THEME because the theme configuration handles specify the clinical-lowlight-theme as the theme that should be the theme used during testing.

```
npm run wdio
```

Today, this will not create a `clinical-lowlight-theme` it will aggregate the lowlight theme as the default and put those screenshots into:

`/wdio/_snapshots_/reference/en `

## Section 2

> Then, they will generate screenshots for the terra-default-theme because their app also supports it:

```
THEME='terra-default-theme` npm run wdio
```

> currently today, this we generate screenshots like:

```
/wdio/_snapshots_/reference/en        <-- no theme folder...prob needs to in next MVB 
```

Today, specifying the theme name as an environment variable will generate a theme folder. Running `THEME='terra-default-theme' npm run wdio` will put the screenshots into:

```
/wdio/_snapshots_/reference/terra-default-theme/en 
```

## Section 3

> Also, neither the ThemePlugin & aggregate-themes script account for the terra-default-theme, which will result in no aggregation...Maybe the Theme Plugin if it didn't filter out the default theme. (@mjhenkes does the fallback get filtered out or just the additional themes?)

Theme aggregation will always run if there is a `terra-theme.config.js` file. The theme plugin doesn't ever actually filter out the default theme because there are no class names to filter through. 

I think it would be non-passive to do now, but I do think for the non-default default scenario (lowlight as default) the screenshots should be put into a lowlight directory. If a theme config file is present (or if we choose to always do this) screenshots should be sorted by the current theme the test is being run in.

Comment (emilyrohrbough) [reply config/webpack/postcss/ThemePlugin.js:11]: > I'm having a little trouble following what you are expecting to happen vs what happens today.

Based on my thoughts & your responses, I think our theming setup in the test harness isn't quite right and could use some improvement.

> Theme aggregation will always run if there is a terra-theme.config.js file. The theme plugin doesn't ever actually filter out the default theme because there are no class names to filter through.

During wdio when we specify the `THEME` env, we override the themeConfig and only aggregate the `THEME` as the default theme: https://github.com/cerner/terra-toolkit-boneyard/blob/master/config/webpack/webpack.config.js#L224

Comment (emilyrohrbough) [reply config/webpack/postcss/ThemePlugin.js:11]: Talked offline...This is next MVB changes
